documentation/src/main/asciidoc/topics/con_eviction_overview.adoc

```diff
@@ -0,0 +1,87 @@
+[id='how_eviction_works-{context}']
+= How Eviction Works
+{brandname} eviction relies on two configurations:
+
+* Size of the data container.
+* Eviction strategy.
+
+.Calculating data container size
+
+You configure the maximum size of the data container and specify if {brandname}
+stores cache entries as:
+
+* Object in the Java heap.
+* Binary `byte[]` in the Java heap.
+* Binary `byte[]` in native memory (off-heap).
+
+|===
+| Storage type | Size of the data container is calculated as:
+
+| Object
+| Number of entries.
+
+| Binary
+| Number of entries, if the eviction type is `COUNT`. +
+Amount of memory, in bytes, if the eviction type is `MEMORY`.
+
+| Off-heap
+| Number of entries, if the eviction type is `COUNT`. +
+Amount of memory, in bytes, if the eviction type is `MEMORY`.
+|===
+
+To calculate the data container size, {brandname} does one of the following:
+
+* Iterates over caches to count the number of entries to determine size.
+* Takes the size of each `byte[]` and includes the overhead of object pointers to determine size.
+
+[NOTE]
+====
+When using `MEMORY`, {brandname} can determine only an approximate size of data containers. For the HotSpot JVM, {brandname} can calculate an approximate size within 3% of the actual size. For other JVMs, the calculation can be within 10%.
+
+When using `MEMORY` with off-heap storage, the calculation is more precise and typically within 1% of the actual size.
```

Comment:
Same here, we may want to just mention that it is a closer approximation that the on heap ones possibly.